staging_service/app.py

```diff
@@ -626,10 +643,15 @@ def inject_config_dependencies(config):
         FILE_EXTENSION_MAPPINGS = os.path.normpath(
             os.path.join(os.getcwd(), FILE_EXTENSION_MAPPINGS)
         )
+    if DTS_MANIFEST_SCHEMA_PATH.startswith("."):
+        DTS_MANIFEST_SCHEMA_PATH = os.path.normpath(
+            os.path.join(os.getcwd(), DTS_MANIFEST_SCHEMA_PATH)
+        )
 
     Path._DATA_DIR = DATA_DIR
     Path._META_DIR = META_DIR
     Path._CONCIERGE_PATH = CONCIERGE_PATH
+    Path._DTS_MANIFEST_SCHEMA_PATH = DTS_MANIFEST_SCHEMA_PATH
```

Comment:
Is having this stored in the Path class still useful?

Reply:
I only left it there to keep the config pattern. But since it's not used, I suppose not.

Reply:
removed.

Reply:
:+1: